File: Yinka Olowofela/Array & Strings/ReverseArray.py
def reverseA(arr):

  arr_len = len(arr) - 1
  
  for i in range(len(arr)//2):
        temp = arr[i]
        arr[i] = arr[arr_len - i]
        arr[arr_len - i] = temp

  return arr

print(reverseA([9,4,6,4,3,4,10,8,9,5,67]))

File: golclinics/arrays-snd-strings/reverseArray.js
const ReverseArray = (a) => {
  let left = 0;
  let right = a.length - 1;
  while (left < right) {
    swap(a, left, right);
    left++;
    right--;
  }
  return a;
};

function swap(arr, left, right) {
  let temp = arr[left];
  arr[left] = arr[right];
  arr[right] = temp;
}

console.log(ReverseArray([1, 4, 5, 6]));
